Nakama/ApiClient.gen.cs

```diff
@@ -4574,9 +4578,12 @@ public ApiClient(Uri baseUri, IHttpAdapter httpAdapter, int timeout = 10)
 
             var method = "POST";
             var headers = new Dictionary<string, string>();
-            var credentials = Encoding.UTF8.GetBytes(basicAuthUsername + ":" + basicAuthPassword);
-            var header = string.Concat("Basic ", Convert.ToBase64String(credentials));
-            headers.Add("Authorization", header);
+            if (!string.IsNullOrEmpty(basicAuthUsername))
```

Comment:
What was the reason behind this? My only concern is that if someone passes in a null or empty server key, the server message will be degraded after this change (e.g., it will probably say something like, there was no auth header rather than a more precise error message about the server key.)

Reply:
I believe this change was introduced in the generator for Satori support. The sever error is unchanced, the API simply returns Unauthorized whether the header is present or not.